app/controllers/fuel_controller.rb

```diff
@@ -14,4 +14,44 @@ def s3_bucket
     )
   end
 
+  def self.paginated_action(options={})
+    before_filter(options) do |controller|
```

Comment:
Suggest determining all pagination attributes here.

Look for the `config.header_pagination` flag, and if present then attempt to use Range. If not present then look for `params[:page]`. If all pagination configs are left undefined then set current page to `1` and per_page to `nil`; I believe that instructs Kaminari to return everything.